src/box/lua/schema.lua

```diff
@@ -108,6 +108,12 @@ ffi.cdef[[
         PORT_C_ENTRY_BOOL,
         PORT_C_ENTRY_MP,
         PORT_C_ENTRY_MP_OBJECT,
+        PORT_C_ENTRY_MP_ITERABLE,
+    };
+
+    struct port_c_iterable {
+        void *iterator_create;
```

Comment:
Nit: Why not use `port_c_iterator_create_f` type here? I think you could define types with `typedef` in Lua FFI.